services/apps/alcs/src/portal/application-submission/application-owner/application-owner.service.ts

```diff
@@ -214,16 +214,26 @@ export class ApplicationOwnerService {
     });
   }
 
-  async deleteAgents(application: ApplicationSubmission) {
+  async deleteNonParcelOwners(application: ApplicationSubmission) {
```

Comment:
maybe I missed it but I do not see unit tests for this

Reply:
Test added!